<!-- UAV: all nodes that launch on UAV (jetson) -->
<launch>

    <!-- vim: set ft=xml noet : -->
    <arg name="fcu_url" default="serial:///dev/ttyUSB0:921600" />
    <arg name="gcs_url" default="" />
    <arg name="tgt_system" default="2" />
    <arg name="tgt_component" default="1" />
    <arg name="log_output" default="screen" />
    <arg name="uav_id" default="1" />
    <arg name="machine_name" default="Quad1" />
    <arg name="pluginlists_yaml" default="$(find mavros)/launch/px4_pluginlists.yaml" />
    <arg name="config_yaml" default="$(find mavros)/launch/apm_config.yaml" />

    <!--

    <param name="image_transport" type="string" value="compressed" />

    <node pkg="machine_vision" type="stereo_driver" name="stereo_cam" machine="$(arg machine_name)" />

    <node pkg="gcs" type="backup_control" name="backup_control"
          machine="$(arg machine_name)" respawn="true">
    </node>

    -->

    <!-- todo node object avoidance-->

</launch>
